address-form/src/app/address-form/address-form.component.html

```diff
@@ -1,70 +1,85 @@
-          <option [ngValue]="c.id" *ngFor="let c of countryList" value="c.name">
-            {{c.name}}
-          </option>
-        </select>
-      </p>
-    </form>
-
-  </div>
-
-  <div style="float: right; width:50%;">
-
-    <h3>Form Status</h3>
-    <b>valid : </b>{{addressForm.valid}}
-    <b>invalid : </b>{{addressForm.invalid}}
-    <b>touched : </b>{{addressForm.touched}}
-    <b>untouched : </b>{{addressForm.untouched}}
-    <b>pristine : </b>{{addressForm.pristine}}
-    <b>dirty : </b>{{addressForm.dirty}}
-    <b>disabled : </b>{{addressForm.disabled}}
-    <b>enabled : </b>{{addressForm.enabled}}
-
-    <h3>Form Value</h3>
-    {{addressForm.value |json}}
-
-  </div>
+<div style="float: left; width:50%;">
+
+  <form [formGroup]="addressForm" (ngSubmit)="onSubmit(addressForm.value)" novalidate action="javascript:;" >
+
+    <p>
+      <label for="streetlineone">Street Line 1 </label>
+      <input type="text" id="streetlineone" name="streetlineone" formControlName="streetOne">
+    </p>
+
+
+    <p>
+      <label for="streetTwo">Street Line 2 </label>
+      <input type="text" id="streetTwo" name="streetTwo" formControlName="streetTwo">
+    </p>
+
+    <p>
+      <label for="town">Town </label>
+      <input type="text" id="town" name="town" formControlName="town">
+    </p>
+
+    <p>
+      <label for="city">City </label>
+      <input type="text" id="city" name="city" formControlName="city">
+    </p>
+
+    <p>
+      <label for="state">State </label>
+      <input type="text" id="state" name="region" formControlName="region">
+    </p>
+
+<!--    <div *ngIf="!isCountrySelected">-->
+<!--      <label for="province">Province </label>-->
+<!--      <input type="text" id="province" name="region" formControlName="region">-->
+<!--    </div>-->
+
+    <p>
+      <label for="postalCode">Postal Code </label>
+      <input type="text" id="postalCode" name="postalCode" formControlName="postalCode">
+    </p>
+
+    <p>
+      <label for="country">Country </label>
+      <select id="country" name="country" formControlName="country">
+        <option [ngValue]="c.name" *ngFor="let c of countryList" value="c.name">
+          {{c.name}}
+        </option>
+      </select>
+    </p>
+
+    <p>
+      <button type="submit">Search</button>
+    </p>
+  </form>
+
+</div>
+
+<div style="float: right; width:50%;">
+
+  <h3>Form Status</h3>
+  <b>valid : </b>{{addressForm.valid}}
+  <b>invalid : </b>{{addressForm.invalid}}
+  <b>touched : </b>{{addressForm.touched}}
+  <b>untouched : </b>{{addressForm.untouched}}
+  <b>pristine : </b>{{addressForm.pristine}}
+  <b>dirty : </b>{{addressForm.dirty}}
+  <b>disabled : </b>{{addressForm.disabled}}
+  <b>enabled : </b>{{addressForm.enabled}}
+
+  <h3>Form Value</h3>
+  {{addressForm.value |json}}
+
+  <h3>Search Results</h3>
+  <ul class="search-result">
+    <li *ngFor="let address of searchResults" >
```

Comment:
This is the easiest way to display the data back as a result for now but we can create a complex loop to show all existing fields of the response.